README.md

```diff
@@ -1,57 +1,113 @@
+    1. Set an environment variable pointing to the file path you just created, here is an example with Powershell and bash:
+        Powershell 
+        ```powershell
+       [Environment]::SetEnvironmentVariable("AZURE_AUTH_LOCATION", "C:\sdksample\azureauth.json", "User")
+       ```
+        Bash
+        ```bash
+        export AZURE_AUTH_LOCATION=/sdksamples/azureauth.json
+        ``` 
+
+
+# What is anf-java-crr-sdk-nfs4.1-sample.dll doing? 
+
+This sample project is dedicated to demonstrate how to enable cross-region replication in Azure NetApp Files for a NFS v4.1 enabled volume, similar to other examples, the authentication method is based on a service principal, this project will create two ANF Accounts in different regions with capacity pool. A single volume using Premium service level tier in the Source ANF, and Data Replication Volume with Standard service level tier in the destination region. 
+
+# How the project is structured
+
+The following table describes all files within this solution:
+
+| Folder         | FileName                    | Description                                                                                                                                                                                                                                                               |
+|----------------|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Root\\^           | main.java                   | Reads configuration, authenticates, executes all operations
+| Root\\^           | Creation.java               | Class that contains all resource creation loops, following the hierarchy logic in order to successfully deploy Azure NetApp Files resources
+| Root\\^           | Cleanup.java                | Class that performs cleanup of all artifacts that were created during this sample application. Its call is commented out by default in main.java
+| Root\\^\common    | CommonSdk.java              | Class dedicated to common operations related to ANF's SDK
+| Root\\^\common    | ResourceUriUtils.java       | Class that exposes a few methods that help parsing Uri's, building new Uri's, or getting a resource name from a Uri, etc
+| Root\\^\common    | ServiceCredentialsAuth.java | A small support class for extracting and creating credentials from a File
+| Root\\^\common    | Utils.java                  | Class that exposes methods that help with getting the configuration object, byte conversion, etc
+| Root\\^\model     | AzureAuthInfo.java          | POJO class to hold authentication information
```

Comment:
What is a POJO Class?

Reply:
Plain old Java object, equivalent to POCO in .NET